robottelo/virtwho_utils.py

```diff
@@ -378,19 +378,23 @@ def hypervisor_json_create(hypervisors, guests):
     :param hypervisors: how many hypervisors will be created
     :param guests: how many guests will be created
     """
-    hypervisors_list = []
     for i in range(hypervisors):
-        guest_list = []
+        hypervisors_list = []
```

Comment:
This is going to reset the list if `hypervisors` is greater than 1, and the resulting dict will only have the last list.

You need to init this list outside of the outermost `for` loop so that it continues to collect dictionaries of guests.